script/build.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Single source of truth for building Firecloud-Orchestration.
+# @ Jackie Roberti
+#
+# Provide command line options to do one or several things:
+#   jar : build orch jar
+#   -d | --docker : provide arg either "build" or "push", to build and push docker image
+# Jenkins build job should run with all options, for example,
+#   ./script/build.sh jar -d push
+
+set -ex
+PROJECT=firecloud-orchestration
+
+function make_jar()
+{
+    # Set for versioning the jar
+    GIT_MODEL_HASH=$(git rev-parse --short ${GIT_BRANCH})
+
+    docker run --rm -e GIT_MODEL_HASH=${GIT_MODEL_HASH} \
+        -v $PWD:/working -w /working -v jar-cache:/root/.ivy -v jar-cache:/root/.ivy2 \
```

Comment:
essentially the first time we run it it creates these volumes (`jar-cache`) and then from then on loads them from the volume.  (this is based on what isaac did in his local scripts)